src/Data/Repositories/ChannelRepository.cs

```diff
@@ -118,7 +118,9 @@ public async Task<(bool, string?)> SafeRemove(Channel type, bool forceClose = fa
             var map = new JobDataMap();
             map.Put("closeRequestId", closeRequest.Id);
             map.Put("forceClose", forceClose);
-            var job = RetriableJob.Create<ChannelCloseJob>(map, closeRequest.Id.ToString());
+
+            var retryList = RetriableJob.ParseRetryListFromEnvironmenVariable("JOB_RETRY_LIST_CHANNEL_CLOSE");
```

Comment:
Why CHANNEL_CLOSE? I mean this retry env var should be for every single retriable job, no matter the context or what it does. Maybe we can use something like JOB_RETRY_INTERVALS, a better name can be found for sure

Reply:
sure, I thought we might want to handle them separately but we can do just one env var for all. I'll do the change

> Quick question, so this would trigger as many times as the lenght of the interval env-var right?

Yes, this would trigger as many times as the length of the interval you set up.